Tasks/DownloadBuildArtifacts/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,20 +1,31 @@
 {
-  "loc.friendlyName": "Download Artifacts - Build",
+  "loc.friendlyName": "Download Build Artifacts",
   "loc.helpMarkDown": "",
-  "loc.description": "Download build artifacts",
-  "loc.instanceNameFormat": "Download Artifacts - Build",
+  "loc.description": "Download Build Artifacts",
+  "loc.instanceNameFormat": "Download Build Artifacts",
+  "loc.input.label.buildType": "Download artifacts produced by",
+  "loc.input.help.buildType": "Download artifacts produced by the current build, or from a specific build.",
   "loc.input.label.project": "Project",
-  "loc.input.help.project": "Select the project name",
+  "loc.input.help.project": "The project from which to download the build artifacts",
   "loc.input.label.definition": "Build definition",
   "loc.input.help.definition": "Select the build definition name",
-  "loc.input.label.version": "Build",
-  "loc.input.help.version": "Select the build number",
-  "loc.input.label.itemPattern": "Item Pattern",
-  "loc.input.help.itemPattern": "Minmatch pattern to filter files to be downloaded.",
+  "loc.input.label.buildId": "Build",
+  "loc.input.help.buildId": "The build from which to download the artifacts",
+  "loc.input.label.downloadType": "Download Type",
+  "loc.input.help.downloadType": "Download specific or multiple artifacts matching pattern.",
+  "loc.input.label.artifactName": "Artifact name",
+  "loc.input.help.artifactName": "The name of the artifact to download",
+  "loc.input.label.itemPattern": "Matching Pattern",
```

Comment:
Do we capitalize multiple-word names like
`Matching Pattern`
or
`Matching pattern`
?

Reply:
I have changed it to small at all places.